benchmarks/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM debian:buster-slim as base
 
-ARG PYTHON_VERSION=3.9.15
+ARG PYTHON_VERSION=3.12.10
```

Comment:
```suggestion
ARG PYTHON_VERSION=3.12
```

Reply:
Why not 3.13? is the performance issue fixed there?

Reply:
I'm open to 3.13, no objection (I sort of forgot about it 😆)

Reply:
My initial thoughts was you probably wanted to be conservative on the version bump but I have no problem with 3.13 as well